src/player_hardcoded_effects.cpp

```diff
@@ -1002,15 +1002,17 @@ void player::hardcoded_effects( effect &it )
 
         // TODO: Move this to update_needs when NPCs can mutate
         if( calendar::once_every( 10_minutes ) && has_trait( trait_id( "CHLOROMORPH" ) ) &&
-            g->is_in_sunlight( pos() ) ) {
+            g->m.is_outside( pos() ) && g->natural_light_level( posz() ) >= 12 && ( g->weather == WEATHER_CLEAR || g->weather == WEATHER_SUNNY
+            || g->weather == WEATHER_DRIZZLE || g->weather == WEATHER_RAINY || g->weather == WEATHER_FLURRIES
+            || g->weather == WEATHER_CLOUDY || g->weather == WEATHER_SNOW ) ) {
             // Hunger and thirst fall before your Chloromorphic physiology!
             if( get_hunger() >= -30 ) {
                 mod_hunger( -5 );
             }
             if( get_thirst() >= -30 ) {
                 mod_thirst( -5 );
             }
-        }
+          }
```

Comment:
This change is unneeded. The closing curly bracket should have the same spaces before it as the opening `if`. I.e. they should be on the same vertical line.

Reply:
Thanks, just sent a new commit for that.

Reply:
I don't know if you noticed but just moving that bracket made it fail even more. and I still don't know what the problem is. the "style" or something.